--- a/config.h	2020-07-21 08:42:19.012656003 +0200
+++ b/config.h	2020-07-21 08:45:35.885002412 +0200
@@ -41,12 +41,12 @@
 #define SMALLDNAME 50 /* most domain names are smaller than this */
 #define CNAME_CHAIN 10 /* chains longer than this atr dropped for loop protection */
 #define DNSSEC_MIN_TTL 60 /* DNSKEY and DS records in cache last at least this long */
-#define HOSTSFILE "/etc/hosts"
-#define ETHERSFILE "/etc/ethers"
+#define HOSTSFILE "/app/config/hosts"
+#define ETHERSFILE "/app/config/ethers"
 #define DEFLEASE 3600 /* default DHCPv4 lease time, one hour */
 #define DEFLEASE6 (3600*24) /* default lease time for DHCPv6. One day. */
-#define CHUSER "nobody"
-#define CHGRP "dip"
+#define CHUSER "app"
+#define CHGRP "app"
 #define TFTP_MAX_CONNECTIONS 50 /* max simultaneous connections */
 #define LOG_MAX 5 /* log-queue length */
 #define RANDFILE "/dev/urandom"
@@ -174,6 +174,10 @@
 #define HAVE_IPSET 
 #define HAVE_LOOP
 #define HAVE_DUMPFILE
+#define LEASEFILE "/app/data/leases"
+#define CONFFILE "/app/config/dnsmasq.conf"
+#define RESOLVFILE "/app/config/resolv.conf"
+#define RUNFILE "/dev/shm/dnsmasq.pid"
 
 /* Build options which require external libraries.
    
@@ -185,9 +189,9 @@
 /* #define HAVE_LUASCRIPT */
 /* #define HAVE_DBUS */
 /* #define HAVE_IDN */
-/* #define HAVE_LIBIDN2 */
-/* #define HAVE_CONNTRACK */
-/* #define HAVE_DNSSEC */
+#define HAVE_LIBIDN2
+#define HAVE_CONNTRACK
+#define HAVE_DNSSEC
 
 
 /* Default locations for important system files. */
